### DISCUSSION TITLE
How to change memory size for minecraft server

### DISCUSSION THREAD

**Discussion by cdoughtonn** [itzg/docker-minecraft-server#2134] [Q&A]
Hi. I'm new to the docker space and trying to make a Minecraft server. I have the server running fine, but I didn't allocate enough memory. I don't know how to use any argument I would need to change the memory size without entirely deleting the container in portainer and doing the initial docker run etc command

**Comment by itzg** [comment]
Re-creating the container is standard practice and the only way to change the environment variables, such as MEMORY in your scenario. Containers are ephemeral, so make use of volume bindings like `/data` in this case to keep your data persistent while swapping out the container.
